flake.nix

```diff
@@ -4,7 +4,7 @@
     common.url = "github:nammayatri/common";
     flake-parts.follows = "common/flake-parts";
 
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/5b2e22593b7e808c0674c444b9cbb30ab0de2922";
```

Comment:
shared-kernel should be updated without changing `flake.nix`; see https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs